src/js/datadog.js

```diff
@@ -6,7 +6,7 @@ import { datadogConfig as config, versions, appConfig as app } from './config';
 
 function initLogs({ isForm }) {
   datadogLogs.init({
-    env: app.env,
```

Comment:
Pretty sure this is the only line that should be in the whole PR

Reply:
Woah. Don't know how that got there.

Reply:
![there i fixed it](https://media.giphy.com/media/VeSvZhPrqgZxx2KpOA/giphy.gif)